Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs

```diff
@@ -106,92 +84,77 @@ public async Task<IEnumerable<RemoteSubtitleInfo>> Search(SubtitleSearchRequest
 
             await Login(cancellationToken).ConfigureAwait(false);
 
-            var subLanguageId = Util.ThreeLetterToTwoLetterISO(request.Language);
-
             string hash;
-            using (var fileStream = File.OpenRead(request.MediaPath))
+            await using (var fileStream = File.OpenRead(request.MediaPath))
             {
                 hash = Util.ComputeHash(fileStream);
             }
 
-            var searchImdbId = request.ContentType == VideoContentType.Movie ? imdbId.ToString(UsCulture) : string.Empty;
-            var name = request.ContentType == VideoContentType.Episode ? request.SeriesName : Path.GetFileName(request.MediaPath);
+            var options = new Dictionary<string, string> { { "languages", request.TwoLetterISOLanguageName }, { "moviehash", hash }, { "type", request.ContentType == VideoContentType.Episode ? "episode" : "movie" }, };
 
-            var p = new Dictionary<string, string>
-            {
-                { "languages", subLanguageId },
-                { "moviehash", hash },
-                // API requires at least 3 letters
-                { "query", name.Length <= 2 ? string.Format(CultureInfo.InvariantCulture, "{0} - {1}", request.ProductionYear, name) : name },
```

Comment:
I found that using the query too usually resulted in no subtitles being found at all when using the hash it has matches. I had better results when I removed the query!

Reply:
yeah I kinda marked that wrong, I'm talking about lines 104 & 110 where query is added, those need length fixes imo

Reply:
It generally only uses the query if there is no IMDb ID, what would you suggest to make it at least two long?

Reply:
in case there is somehow no imdb id it will use movie name as query, and if the movie name is 2 or less characters long API will respond with error

```json
{
    "errors": [
        "Query is too short"
    ],
    "status": 400
}
```

which is why I made it [add production year in case length is <= 2](https://github.com/MBR-0001/jellyfin-plugin-opensubtitles/blob/master/Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs#L125)